docs/data/base/getting-started/overview/overview.md

```diff
@@ -0,0 +1,23 @@
+# MUI Base - Overview
+
+<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks, ideally suited for implementing a custom design system.</p>
+
+## Introduction
+
+MUI Base is library of unstyled React UI components.
+It features the same robust engineering as Material UI, but without implementing Material Design.
+
+With MUI Base, developers can rapidly build UIs with custom design systems, without the need to override any default style settings.
+
+## Why use MUI Base?
+
+MUI Base is ideally suited for developers working with a custom design system.
+
+The library includes prebuilt, unstyled components with production-ready functionality, along with low-level hooks for transferring that functionality to other components.
+
+### MUI Base vs. Material UI
+
+MUI Base features many of the same components as Material UI, but _without the Material Design implementation_.
+
+You could think of MUI Base as the "skeletal" version of Material UI—foundational components with no built-in style choices.
+This makes MUI Base a better option than Material UI if you need to customize the design significantly beyond the Material Design specifications.
```

Comment:
Similar to the comment above, I think Base is great if you want to own CSS entirely. You already can stray away significantly from Material Design with Material UI but it definitely demands experience with the theme structure. With Base, however, there's no such thing as a default theme, you're free to do CSS as you wish.

Reply:
That's a good way of putting it!